doc/nrf/ug_bootloader.rst

```diff
@@ -92,7 +92,94 @@ These signed variants can be used to perform an upgrade procedure through the :r
 Adding a bootloader chain to your application
 *********************************************
 
-See :ref:`bootloader_build_and_run` in the documentation of the :ref:`bootloader` sample for instructions on how to add an immutable bootloader as first stage of the secure bootloader chain.
+The |NCS| includes a sample implementation of an immutable bootloader.
+As solution for implementing an upgradable bootloader, it provides and extends :doc:`mcuboot:index`.
```

Comment:
No idea why I can't answer to https://github.com/nrfconnect/sdk-nrf/pull/2325#discussion_r435085969
What I'm trying to state in these two sentences is what solutions the NCS provides for the bootloader chain. For the immutable bootloader, there is a sample. For the upgradable bootloader, there is our fork of MCUboot (which is an extension of the "real" MCUboot).

Reply:
```suggestion
Additionally, |NCS| comes with a slightly modified version of :doc:`mcuboot:index`.
```

Reply:
Almost :)
```suggestion
Additionally, the |NCS| comes with a slightly modified version of :doc:`mcuboot:index`.
```